src/i18n/en-US.properties

```diff
@@ -67,29 +67,31 @@ file_preview_label=File preview
 
 # Error messages
 # Default preview error message
-error_generic=We’re sorry, the preview didn’t load.
+error_generic=Preview did not load. Open or download the file to view.
 # Default preview error message
-error_unsupported=We’re sorry, we can’t preview this file type in your web browser. To view this content please download and open it on your device.
+error_unsupported=File type unsupported for preview. Open or download the file to view.
 # Account doesn’t have a sufficient tariff to preview the requested file type
-error_account=We’re sorry, your account is unable to preview this file type.
+error_account=We’re sorry, your account is unable to preview this file type. Open or download the file to view.
 # No permissions preview error message
 error_permissions=We’re sorry, you don’t have permission to preview this file.
-# Preview refresh error message suggesting refreshing the page as a possible fix
-error_refresh=We’re sorry, the preview didn’t load. Please refresh the page.
```

Comment:
The messages seem very inconsistent. Some have conciliatory language (e.g. "We're sorry") and some don't. Some have personalized language (e.g. "Your browser") and some don't. We should standardize the form and tone holistically before merging.

Reply:
Hi @jstoffan,

I’ve updated the messages with input from the product team to ensure they are more standardized.